test/ad_localize_test.rb

```diff
@@ -55,15 +56,46 @@ def teardown
       generated_file = "exports/#{file}"
       assert(File.exist?(generated_file), "File does not exists #{generated_file}")
     end
+    FileUtils.rm_rf('exports')
+  end
+
+  test 'it should export multiple input csv files' do
+      # Given
+      csv_files = ['test/reference.csv', 'test/reference1.csv', 'test/reference2.csv']
+      csv_files.each do |file|
+        assert(File.exist?(file), "File does not exists #{file}")
+      end
+      reference_dir = "test/exports_reference_multi"
+      assert(File.exist?(reference_dir), "File does not exists #{reference_dir}")
+      reference_subdirs = [
+        "reference",
+        "reference1",
+        "reference2"
+      ]
+
+      # When
+      runner = AdLocalize::Runner.new
+      runner.run csv_files
+
+      # Then
+      reference_subdirs.each do |subdir|
+          all_files.each do |file|
+            reference_file = "#{reference_dir}/#{subdir}/#{file}"
+            generated_file = "exports/#{subdir}/#{file}"
+            assert(File.exist?(generated_file), "File does not exists #{generated_file}")
+            diff = Diffy::Diff.new(reference_file, generated_file, :source => 'files')
+            assert_empty(diff.to_s, "File #{generated_file} do not match reference. Diff: \n\n#{diff}\n")
+          end
+      end
   end
 
   private
 
-  def all_files
-    ios_files + android_files + json_files + yml_files
+  def all_files(languages: ["fr", "en"])
```

Comment:
Can you set the languages in the setup method ?

Reply:
Langage should be set for each test not for the whole test suite. 
Tests are broken and not really extensible. 
I was trying to add a simple csv file only in french (with no plurals for example) and tests were failing because of langages being static and file to compare also being static.

Reply:
It is okay to keep the languages parameter. Can you set the language list in the setup method ? If we want to check other languages we would have to update every method signature and we want to avoid that